src/highdicom/sr/enum.py

```diff
@@ -4,77 +4,199 @@
 
 class ValueTypeValues(Enum):
 
-    """Enumerated values for attribute Value Type."""
+    """Enumerated values for attribute Value Type.
+
+    See :dcm:`Table C.17.3.2.1 <part03/sect_C.17.3.2.html#sect_C.17.3.2.1>`.
+
+    """
 
     CODE = 'CODE'
+    """Coded expression of the concept."""
+
     COMPOSITE = 'COMPOSITE'
+    """Reference to UIDs of Composite SOP Instances."""
+
     CONTAINER = 'CONTAINER'
+    """The content of the CONTAINER. The value of a CONTAINER Content Item is
+    the collection of Content Items that it contains."""
+
     DATE = 'DATE'
+    """Calendar date."""
+
     DATETIME = 'DATETIME'
+    """Concatenated date and time."""
+
     IMAGE = 'IMAGE'
+    """Reference to UIDs of Image Composite SOP Instances."""
+
     NUM = 'NUM'
+    """Numeric value and associated Unit of Measurement."""
+
     PNAME = 'PNAME'
+    """Name of person."""
+
     SCOORD = 'SCOORD'
+    """Listing of spatial coordinates."""
+
     SCOORD3D = 'SCOORD3D'
+    """Listing of spatial 3D coordinates."""
+
     TCOORD = 'TCOORD'
+    """Listing of temporal coordinates."""
+
     TEXT = 'TEXT'
+    """Textual expression of the concept."""
+
     TIME = 'TIME'
+    """Time of day."""
+
     UIDREF = 'UIDREF'
+    """Unique Identifier."""
+
     WAVEFORM = 'WAVEFORM'
+    """Reference to UIDs of Waveform Composite SOP Instances."""
 
 
 class GraphicTypeValues(Enum):
 
-    """Enumerated values for attribute Graphic Type."""
+    """Enumerated values for attribute Graphic Type.
+
+    See :dcm:`C.18.6.1.1 <part03/sect_C.18.6.html#sect_C.18.6.1.1>`.
+
+    """
 
     CIRCLE = 'CIRCLE'
+    """A circle defined by two (column,row) pairs. The first point is the
+    central pixel. The second point is a pixel on the perimeter of the
+    circle."""
```

Comment:
```suggestion
    """A circle defined by two (column,row) pairs.
    
    The first pair is the central point and
    the second pair is a point on the perimeter of the circle.
    """
```

Reply:
Let's try to get towards [PEP 257](https://www.python.org/dev/peps/pep-0257/) compliance and use a single-line summary at the top of a longer docstring.